app/src/androidTest/java/org/simple/clinic/help/HelpSyncAndroidTest.kt

```diff
@@ -43,11 +43,7 @@ class HelpSyncAndroidTest {
   @Test
   fun when_pulling_help_from_the_server_it_should_save_the_content_as_a_file() {
```

Comment:
Good point, done. I renamed it to `when_pulling_reports_from_the_server_it_should_save_the_reports` so that the implementation detail of it being a database is hidden.